install_config/install/deploy_router.adoc

```diff
@@ -269,9 +280,63 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be grouped to distribute routing load in the cluster and separate
+tenants to different routers.  Each router in the group shardes a set of routes that are
+tagged using labels. Router sharding over the whole cluster uses ROUTE_LABELS.
```

Comment:
I'd call this one "user-driven router sharding" vs. "administrator-driven route sharding".